src/lib/FlightTasks/tasks/Orbit/FlightTaskOrbit.hpp

```diff
@@ -104,6 +104,9 @@ class FlightTaskOrbit : public FlightTaskManualAltitudeSmooth
 	const float _velocity_max = 10.f;
 	const float _acceleration_max = 2.f;
 
+	uint8_t _yaw_behavior = 0;
```

Comment:
I know it's nit-picking but could you add variable description comments?
I'm a bad example and should fill the others too 😬